client/driver/rkt.go

```diff
@@ -149,6 +149,19 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 		cmd_args = append(cmd_args, fmt.Sprintf("--exec=%v", exec_cmd))
 	}
 
+	if task.Resources.MemoryMB == 0 {
```

Comment:
@jippi Drivers that support resource control, shouldn't allow infinite resources in my opinion since from the point of view of the scheduler finite resources were being assigned to a task.